From 44ca78d0ae968d42803653ef395ce34909f557c6 Mon Sep 17 00:00:00 2001
Message-Id: <44ca78d0ae968d42803653ef395ce34909f557c6@dist-git>
From: Andrea Bolognani <abologna@redhat.com>
Date: Tue, 11 Aug 2015 17:16:03 +0200
Subject: [PATCH] tests: Temporarily disable ppc64 cpu tests

The upcoming commits will make heavy modifications to the ppc64
driver, split so that it's easier to review the changes.

Instead of updating the test cases so that they pass, possibly
only to update them again with the following commit, disable them
for the time being.

Another commit will update them all in one go once all required
changes are in place.

(cherry picked from commit 04f5a60d4b3354edb44a3597e574633fc28eb2d4)

Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1250977

Signed-off-by: Andrea Bolognani <abologna@redhat.com>
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 tests/cputest.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/tests/cputest.c b/tests/cputest.c
index 1e84fd3..4dbccfd 100644
--- a/tests/cputest.c
+++ b/tests/cputest.c
@@ -501,7 +501,9 @@ static const char *model486[]   = { "486" };
 static const char *nomodel[]    = { "nomodel" };
 static const char *models[]     = { "qemu64", "core2duo", "Nehalem" };
 static const char *haswell[]    = { "SandyBridge", "Haswell" };
+/* XXX temporarily disabled
 static const char *ppc_models[] = { "POWER7", "POWER7_v2.1", "POWER7_v2.3", "POWER8_v1.0"};
+*/
 
 static int
 mymain(void)
@@ -595,8 +597,10 @@ mymain(void)
     DO_TEST_COMPARE("x86", "host-worse", "nehalem-force", VIR_CPU_COMPARE_IDENTICAL);
     DO_TEST_COMPARE("x86", "host-SandyBridge", "exact-force-Haswell", VIR_CPU_COMPARE_IDENTICAL);
 
+    /* XXX temporarily disabled
     DO_TEST_COMPARE("ppc64", "host", "strict", VIR_CPU_COMPARE_IDENTICAL);
     DO_TEST_COMPARE("ppc64", "host", "exact", VIR_CPU_COMPARE_INCOMPATIBLE);
+    */
 
     /* guest updates for migration
      * automatically compares host CPU with the result */
@@ -625,8 +629,11 @@ mymain(void)
     DO_TEST_BASELINE("x86", "7", 0, 0);
     DO_TEST_BASELINE("x86", "8", 0, 0);
 
+    /* XXX temporarily disabled
     DO_TEST_BASELINE("ppc64", "incompatible-vendors", 0, -1);
     DO_TEST_BASELINE("ppc64", "no-vendor", 0, 0);
+    */
+
     /* CPU features */
     DO_TEST_HASFEATURE("x86", "host", "vmx", YES);
     DO_TEST_HASFEATURE("x86", "host", "lm", YES);
@@ -661,8 +668,10 @@ mymain(void)
     DO_TEST_GUESTDATA("x86", "host-Haswell-noTSX", "Haswell-noTSX",
                       NULL, "Haswell-noTSX", 0);
 
+    /* XXX temporarily disabled
     DO_TEST_GUESTDATA("ppc64", "host", "guest", ppc_models, NULL, 0);
     DO_TEST_GUESTDATA("ppc64", "host", "guest-nofallback", ppc_models, "POWER7_v2.1", -1);
+    */
 
     return ret == 0 ? EXIT_SUCCESS : EXIT_FAILURE;
 }
-- 
2.5.0

